packages/docs/content/components/drawer.mdx

```diff
@@ -147,24 +147,50 @@ This component has analytics tracking available. Please see our developer docume
 
 #### Focus Management
 
-The property `hasFocusTrap` is an available option that will prevent keyboard focus from leaving the confines of the HelpDrawer when enabled.
+- Focus should move to the content in the Drawer after the user has opened it.
+- The property `hasFocusTrap` is an available option that will prevent keyboard focus from leaving the confines of the Drawer when enabled.
 
-When the property `hasFocusTrap` is enabled...
+When `hasFocusTrap` is enabled...
 
-- Keyboard focus is trapped within the HelpDrawer component.
-- Pressing the escape key will close the Drawer.
+- Keyboard focus is trapped within the Drawer component.
+- Pressing <kbd>Escape</kbd> will close the Drawer.
 
-A focus trap should _not_ be used when it's anticipated that the user will rely on the Drawer as a reference resource for doing something outside of the Drawer. For instance, if the user is completing a form, Drawer may contain a reference guide defining specific requirements.
+A focus trap should _not_ be used when it's anticipated that the user will rely on the Drawer as a reference resource for doing something outside of the Drawer. For instance, if the user is completing a form, Drawer may contain a reference guide defining specific requirements on the page.However, there are times when it makes sense to trap focus within the Drawer. One occasion where it can benefit the user is when the Drawer is anticipated to fill the viewport of a screen, e.g., in a mobile layout.
 
-However, there are times when it makes sense to trap focus within the Drawer. One occassion where it can benefit the user is when the Drawer is anticipated to fill the viewport of a screen, i.e., mobile layout.
+#### Accessibility testing
+
+##### Keyboard
+
+- I can use <kbd>Tab</kbd> to move focus to the Drawer toggle button.
+- I can use <kbd>Enter</kbd> or <kbd>Space</kbd> to toggle the Drawer open or closed.
+- When the Drawer is open, I can use <kbd>Tab</kbd> to move focus to the close button, then sequentially through the list of focusable items.
+- There's a visible focus ring as I move through interactive content in the Drawer, first to the close button, then sequentially through the list of focusable items.
+- If `hasFocusTrap` is enabled, then...
+  - ...pressing <kbd>Escape</kbd> will close the Drawer.
+  - ...I can't move focus outside of the Drawer.
+- If `hasFocusTrap` is not set, then...
+  - ...I can move focus outside of the Drawer.
+
+##### Screen reader
+
+- The Drawer toggle indicates the purpose of the Drawer and mirrors the first heading inside the Drawer.
+- When the Drawer is open:
+  - Focus is sent to the Drawer.
+  - The Drawer's role of `dialog` is read.
+  - The Drawer's first heading element is announced.
+
+###### Mobile
```

Comment:
Whoa, I've never seen us use an h6. Looking forward to some IA updates in the future